Src/Witsml/Data/WitsmlWellboreGeometry.cs

```diff
@@ -0,0 +1,44 @@
+using System.Xml;
+using System.Xml.Serialization;
+using Witsml.Data.Measures;
+
+//Hvor kommer betydning av minOccurs inn?
```

Comment:
minOccurs means that an attribute is required in the query when creating or copying the file. We anyway set most attributes as "optional" because not all queries require the "minOccurs=1" attributes.